lib/twitter-ads/creative/app_download_card.rb

```diff
@@ -12,23 +12,24 @@ class AppDownloadCard
 
       attr_reader :account
 
-      property :id, read_only: true
-      property :preview_url, read_only: true
-      property :deleted, type: :bool, read_only: true
+      property :card_type, read_only: true
+      property :card_uri, read_only: true
       property :created_at, type: :time, read_only: true
+      property :deleted, type: :bool, read_only: true
+      property :id, read_only: true
       property :updated_at, type: :time, read_only: true
 
-      property :name
       property :app_country_code
+      property :app_cta
+      property :custom_icon_media_id
+      property :custom_app_description
       property :iphone_app_id
       property :iphone_deep_link
       property :ipad_app_id
       property :ipad_deep_link
       property :googleplay_app_id
       property :googleplay_deep_link
-      property :app_cta
-      property :custom_icon_media_id
-      property :custom_app_description
+      property :name
 
       RESOURCE_COLLECTION = "/#{TwitterAds::API_VERSION}/" +
                             'accounts/%{account_id}/cards/app_download'.freeze # @api private
```

Comment:
Since we removed the ability to create or update app download cards ([announcement](https://twittercommunity.com/t/deprecation-announcement-app-download-cards/87807)) and don't even include it in the [documentation](https://developer.twitter.com/en/docs/ads/creatives/api-reference), should we remove this class altogether?